src/components/display/icon.tsx

```diff
@@ -0,0 +1,29 @@
+import { FunctionComponent } from 'react';
+
+import { ColorTypes } from '../../themes/theme.types';
+
+import {
+  iconFillMap, IconName, iconsMap, iconStrokeMap,
+} from './icon.types';
+
+interface IconProps {
+  iconName: IconName;
```

Comment:
do we want to be able to control the role from outside, for potential accessibility purposes?

Reply:
Yes, but if we do we'll want to be very careful about not using it for evil. Will add.